packages/node/src/BacktraceClient.ts

```diff
@@ -40,6 +46,7 @@ export class BacktraceClient extends BacktraceCoreClient<BacktraceConfiguration>
             breadcrumbsManager.setStorage(
                 FileBreadcrumbsStorage.create(
                     this.sessionFiles,
+                    fileSystem,
```

Comment:
it should be taken from clientSetup if possible
```suggestion
                    clientSetup?.fileSystem ?? fileSystem,
```

Reply:
Fixed.